OFL.txt

```diff
@@ -1,5 +1,4 @@
-Copyright © 2012-2017 The Libertinus Project Authors, with Reserved Font Name
-"Linux Libertine", "Biolinum", "STIX Fonts".
+Copyright © 2012-2020 The Libertinus Project Authors
```

Comment:
I believe that the reversed font name clause should be kept even of this fork is no longer using this name, since anyone forking it can’t use any of these reserved name.
https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL-FAQ_web&_sc=1#726d6952

Reply:
Thanks for this correction, I went to the FAQ looking for this and apparently missed it.